css/index.css

```diff
@@ -1,3 +1,334 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
 
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+body {
+  margin-left: 20%;
+  margin-right: 20%;
+  margin-top: 2%;
+  font-size: 14px;
+}
+
+.header {
+  display: flex;
+}
+
+a {
+  color: grey;
+  font-size: 12px;
+  text-decoration: none;
+  margin: 22px;
+}
+
+a:hover {
+  
+}
+
+h1 {
+  font-size: 75px;
+  text-align: center;
+}
+
+button {
+  background: white;
+  border: 1px solid black;
+  font-size: 10.5px;
+  width: 100px;
+  height: 20px;
+}
+
+p {
+  margin-bottom: 15px;
+}
+.header{
+    display: flex;
+}
+
+.head {
+  display: inline-flex;
+  margin-bottom: 5%;
+}
+
+.title-pic  {
+  display: flex;
+}
+.top {
+/*   display: inline-block; */
+  width: 49%;
+  text-align: center;
+}
+
+.feature-about  {
+  display: flex;
+}
+.top-mid {
+/*   display: inline-block; */
+  width: 49%;
+  padding: 1px;
+  margin-top: 15px;
+  margin-bottom: 5%;
+}
+
+.middle-img {
+  display: flex;
+  width: 100%;
+}
+.bottom-text  {
+  display: flex;
+}
+.bottom-mid {
+/*   display: inline-block; */
+  width: 93%;
+  margin-top: 10%;
+}
+.contact    {
+    margin-top: 20px;
+}
+
+#copyright {
+  text-align: center;
+}
+@media(max-width: 800px){
+    
```

Comment:
Good job finding and targeting only the HTML elements you needed, and modifying only the properties that you needed to on them. This will make tracking, and modifying properties easier than had you not.